src/components/WalletModal/Option.tsx

```diff
@@ -62,38 +60,163 @@ const IconWrapper = styled.div`
     align-items: flex-end;
   `};
 `
+const WCv2PopoverContent = styled(ThemeButton)`
+  background: ${({ theme }) => theme.backgroundSurface};
+  border: 1px solid ${({ theme }) => theme.backgroundOutline};
+  border-radius: 12px;
+  cursor: pointer;
+  display: flex;
+  max-width: 240px;
+  padding: 16px;
+  position: absolute;
+  z-index: ${Z_INDEX.popover};
+`
+const WCv2PopoverToggle = styled.button`
+  align-items: center;
+  background-color: transparent;
+  border: none;
+  color: ${({ theme }) => theme.textTertiary};
+  cursor: pointer;
+  display: flex;
+  height: 24px;
+  justify-content: center;
+  margin: 0;
+  max-width: 48px;
+  padding: 0;
+  position: absolute;
+  right: 16px;
+  top: calc(50% - 12px);
```

Comment:
does this 12px relate to any of the other properties? if so i like to math them together with a variable so changing one changes the other

Reply:
Here's what that looks like. I like the principle, but in this case it feels a bit cludgy to me. 
```
const TOGGLE_SIZE = 24
const WCv2PopoverToggle = styled.button`
//...
  height: ${TOGGLE_SIZE}px;
//...
  top: calc(50% - ${TOGGLE_SIZE / 2}px);
  width: ${TOGGLE_SIZE}px;
//...
`
```